app/views/task/edit.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="edit">
```

Comment:
I think you could DRY up this view and the new view by creating a partial in the future. Also, watch your indentation in this file as well.